cfgov/jinja2/v1/_includes/molecules/hero.html

```diff
@@ -27,36 +27,36 @@
    value.background_color: A string containing the color code (in any format)
                            to set as the hero background color.
 
-   value.is_white_text:    A boolean to change the text color to white.
+   value.photo:            A boolean to change the hero to an overlay hero.
```

Comment:
I don't know the context of these updates, but some schools of thought would say to prefix boolean variables with `is_` or `has_` so that when reading the variable name one can answer `true` or `false` in their head and have a clearer notion that it's a boolean value. Gotta say `photo` wouldn't strike me as a boolean at first blush. Would we want to say something like `has_overlay_photo`?

Reply:
Yeah, this was an interesting thing about this task. The variable names here are also what shows up in the Wagtail admin view (i.e. the screenshot above), so they have to fit in both code and English. Designers specifically asked for these variable names because they would make sense to content editors and align with the guidelines in the Design Manual.

I decided to make the variable names fit the designers' request rather than force them to use language that was unnatural to them. Since this is the only place where these variables are used, and they're clearly documented, I didn't think this was a risk. If you do think this will make the template harder to use/update, I can reconsider that decision.

Reply:
You can pass in a `label` argument to the field definitions to customize what appears in the Wagtail admin. For example, you could do:

```python
is_bleeding = blocks.BooleanBlock(
    required=False,
    label='Bleed',
    help_text… 
```

Reply:
Awesome. That makes this PR drastically simpler. I just pushed a new version that resolves this.